[2025-09-09T15:32:01.169+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-09T15:32:01.279+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.load manual__2025-09-09T15:26:44.120550+00:00 [queued]>
[2025-09-09T15:32:01.303+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.load manual__2025-09-09T15:26:44.120550+00:00 [queued]>
[2025-09-09T15:32:01.304+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2025-09-09T15:32:01.342+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): load> on 2025-09-09 15:26:44.120550+00:00
[2025-09-09T15:32:01.367+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'load', 'manual__2025-09-09T15:26:44.120550+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmp1y34xjzm']
[2025-09-09T15:32:01.374+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask load
[2025-09-09T15:32:01.377+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=223) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-09T15:32:01.380+0000] {standard_task_runner.py:72} INFO - Started process 224 to run task
[2025-09-09T15:32:01.657+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.load manual__2025-09-09T15:26:44.120550+00:00 [running]> on host 0a8d8de33147
[2025-09-09T15:32:02.087+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-09-09T15:26:44.120550+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-09T15:26:44.120550+00:00'
[2025-09-09T15:32:02.089+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-09T15:32:02.121+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-09T15:32:02.124+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/load.py']
[2025-09-09T15:32:02.153+0000] {subprocess.py:86} INFO - Output:
[2025-09-09T15:32:05.716+0000] {subprocess.py:93} INFO - Loaded to DB
[2025-09-09T15:32:05.949+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-09T15:32:06.144+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-09T15:32:06.146+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=load, run_id=manual__2025-09-09T15:26:44.120550+00:00, execution_date=20250909T152644, start_date=20250909T153201, end_date=20250909T153206
[2025-09-09T15:32:06.229+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-09T15:32:06.352+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-09T15:32:06.376+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-09-09T15:32:06.387+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
